www/www-static.yml

```diff
@@ -29,6 +18,20 @@
     when: not _phpbbforum_style_file_st.stat.exists
 
   tasks:
+  - name: install deployment script
+    become: yes
+    template:
+      src: files/deploy
+      dest: /usr/local/bin/deploy
+      mode: 0555
+
+  - name: run deployment script
+    become: yes
+    command: /usr/local/bin/deploy -f
```

Comment:
Anyway to have this pass `-n` when in check mode?

Reply:
There's an example of doing this in Ansible here:
https://github.com/ome/ansible-role-omero-web/blob/f5056439ea2ea4311c89c7a5177ad5366b144d3a/tasks/web-install-py3.yml#L121-L129

Reply:
That's a good idea - c4ecd86. And tested in and outside check-mode on `ome-www-dev`